modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -1632,7 +1632,10 @@ m|dbms.cluster.setAutomaticallyEnableFreeServers(autoEnable :: BOOLEAN)
 m|WRITE
 |===
 
-
+[NOTE]
+====
+Before Neo4j 5.23, the procedure `dbms.cluster.setAutomaticallyEnableFreeServers()` can be run by users with the `Admin` privileges only.
+====
```

Comment:
Suggestion for more detail:
```suggestion
====
Before Neo4j 5.23, the procedure `dbms.cluster.setAutomaticallyEnableFreeServers()` can be run by users with the `Admin` privileges only.
Since Neo4j 5.23, it can be run with the `SERVER MANAGEMENT` privilege.
It will still run with the `Admin` privilege, but that should be considered deprecated.
====
```

Reply:
(This is true for all three procedures)